README.md

```diff
@@ -1,12 +1,26 @@
 # Vulcanomicon
 
-The Vulcan Academy's operating system
+The Vulcan Academy's operating system.
 
-1. The goal of the game is to create a learning organization
-1. All players must unanimously agree to all rule changes
-1. Proposals may add, amend or repeal a rule
-1. All rules should be logically self-consistent 
+1. The goal of the game is to create a learning organization.
+1. All players must unanimously agree to all rule changes.
+1. Proposals may add, amend or repeal a rule.
+1. All rules should be logically self-consistent. 
+
+## Reality
+1. Our understanding of __Reality__ is throttled by our ability to percieve it. 
+    * We use logic to describe our understanding of __Reality__.
+1. __Reality__ is exclusively inhabited by __Information__, and patterns thereof.
+    * __Information__ contained within __Empirical Reality__ inhabits 3 spatial dimensions and 1 monotonically increasing temporal dimension.
+1. __Information__ may interact with and modify other __Information__. This is an __Action__.
 
 ## Lexicon
 
-agent: any system that is best characterized as acting according to beliefs
\ No newline at end of file
+* __Reality__: The set of everything that exists or does not exist.
```

Comment:
What is an example of something that is real (part of reality) that does not exist?

Reply:
It is a __Reality__ that flying pigs do not exist empirically. The reasoning was that something _not_ existing is just as "real" as something existing.

I meant to have __Reality__ be an architectural handle on the "set of all things". Though maybe this "set of all things" could fall under a different lexeme, and __Reality__ could just be the set of all things that exist if that makes more sense (We would then logically also introduce __Non-Reality__).